.github/workflows/master-pipeline.yml

```
@@ -163,12 +163,11 @@ jobs:
       - name: Build image and push to GitHub Packages
         uses: docker/build-push-action@v6
         with:
-          context: .
-          file: ./Dockerfile
           push: true
-          pull: true
           no-cache: true
+          repository: giveth/impact-graph
```

actionlint (1.7.8)

input "repository" is not defined in action "docker/build-push-action@v6". available inputs are "add-hosts", "allow", "annotations", "attests", "build-args", "build-contexts", "builder", "cache-from", "cache-to", "call", "cgroup-parent", "context", "file", "github-token", "labels", "load", "network", "no-cache", "no-cache-filters", "outputs", "platforms", "provenance", "pull", "push", "sbom", "secret-envs", "secret-files", "secrets", "shm-size", "ssh", "tags", "target", "ulimit"

(action)